api/src/db/migrations/v145_remove_nonprofit_onboarding.ts

```diff
@@ -0,0 +1,805 @@
+import { v144Business, v144UserData } from "@db/migrations/v144_add_raffle_bingo_games";
+import { randomInt } from "@shared/intHelpers";
+
+export const migrate_v144_to_v145 = (v144Data: v144UserData): v145UserData => {
+  return {
+    ...v144Data,
+    businesses: Object.fromEntries(
+      Object.values(v144Data.businesses)
+        .map((business: v144Business) => migrate_v144Business_to_v145Business(business))
+        .map((currBusiness: v145Business) => [currBusiness.id, currBusiness])
+    ),
+    version: 145,
+  } as v145UserData;
+};
+
+export const migrate_v144Business_to_v145Business = (business: v144Business): v145Business => {
+  return {
+    ...business,
+    profileData: {
+      ...business.profileData,
+      isNonprofitOnboardingRadio: undefined,
```

Comment:
I considered writing a test here and even started but found the typing throwing up at us. Plus it felt more like adding a field in the sense that it feels redundant to check a value that literally doesn't matter or do anything at the moment.